spec/support/work_packages/pdf_export.rb

```
@@ -9,6 +9,10 @@ def column_title(column_name)
   end
 
   def label_title(column_name)
+    if column_name.start_with?("cf_")
+      id = column_name.delete_prefix("cf_").to_i
+      return ::CustomField.find_by(id:).name
```

Using `find_by` prevents ActiveRecord from raising an error when the record could not be found. Instead it returns `nil`. This means that calling `#name` on `nil` will raise a `NoMethodError`, which is very unspecific and a pain to debug when it happens.

I suggest:
* Either use `::CustomField.find(id)` instead. It will raise an error if the custom field cannot be found. This is the way to go if a missing custom field should prevent the export.
* Or keep `find_by`, but use some kind of placeholder name in case the custom field does not exist, for example: `::CustomField.find_by(id:)&.name || "default name"`. This is the preferred option if we want an export to succeed in such a case.

I _think_ it would be nice to enable the export and fall back to a default value. It's better to have an incomplete export than having none at all 😅 But then again I am not an avid user of exports, so take that opinion with a grain of salt.

---

Thanks for the tip! I'm using `::CustomField.find(id)` now. This is just a helper function in the specs, so it should throw a proper error, and a prevented export is fine!

---

Oh, duh I didn't notice the file path. Then it's an easy one! :)